tests/data/md_helium/berendsen_barostat.toml

```diff
@@ -0,0 +1,20 @@
+[input]
+version = 1
+
+[[systems]]
+file = "../configuration/helium.xyz"
+cell = 10
+velocities = {init = "300 K"}
+
+[[systems.potentials.pairs]]
+atoms = ["He", "He"]
+lj = {sigma = "2 A", epsilon = "0.2 kJ/mol"}
+cutoff = "10 A"
+
+[[simulations]]
+nsteps = 1_000
+
+[simulations.propagator]
+type = "MolecularDynamics"
+timestep = "1 fs"
+integrator = {type = "BerendsenBarostat", pressure = "5000 bar", timestep = 1000}
```

Comment:
This simulation need a thermostat too, the same as in the other comment should do.